modules/Sources/Models/AddressBookEncryptionKeys.swift

```diff
@@ -6,18 +6,27 @@
 //
 
 import Foundation
+import CryptoKit
 
 /// Representation of the address book encryption keys
-public struct AddressBookEncryptionKeys: Codable, Equatable {
-    public let key: String
+public struct AddressBookEncryptionKeys: Equatable, Decodable, Encodable {
+    public let key: SymmetricKey?
 
-    public init(key: String) {
+    public init(key: SymmetricKey?) {
         self.key = key
     }
+    
+    public init(from decoder: any Decoder) throws {
+        self.key = nil
+    }
+    
+    public func encode(to encoder: any Encoder) throws {
+        
```

Comment:
```suggestion
        // encryption keys should not be encoded
```

Reply:
We need to be able to encode the key in some fashion in order to cache it separately from the seed. IDK if this is actually where we want to encode it though, per my other todo.